grafana/cluster/base/subscription.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   name: grafana-operator
   namespace: opendatahub
 spec:
-  channel: alpha
+  channel: v4
   installPlanApproval: Automatic
```

Comment:
Does channel change still work on older OCP versions, ie OCP 4.5,4.6,4.7,4.8?

Reply:
Yes, there is no version restriction in the CSV (https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/grafana-operator/4.0.2/grafana-operator.clusterserviceversion.yaml).

I tested 4.9 in my cluster and 4.7 through CI.